azure-pipelines.yml

```diff
@@ -34,6 +34,7 @@ stages:
     jobs:
     - template: run-tox-env.yml@OpenAstronomy
       parameters:
+        xvfb: true
```

Comment:
Does this not need to be set in  the second stage as well?

Reply:
We're only building docs right?
For now we're skipping the examples for docs right so I think it should be fine right?
We can add them in when the git-hub workflows are done?

Reply:
Sure.